### PR TITLE
feat(selection-page): add localForage, projects History

### PR DESCRIPTION
**Pull request by Teyz**
In this PR : 
- Homepage now has the list of projects already open
- 4 projects max in the list, if you select an other project, the first project will change
- All the logic of localForage is on composable localForage

The quality of the code can be improved. This is a first draft.

### CODE REVIEW

File: src/composables/localForage.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+const handleHistoryLenght = (projectsHistory, project) => {
+  if (projectsHistory.length === 4) {
+    projectsHistory[0] = project;
+  } else {
+    projectsHistory.push(project);
+  }
+};
+
+export const localForage = () => ({
```
Comment (felixbouveret) [inline src/composables/localForage.js:40]: To stick with the composition API good practices
```suggestion
export const useLocalForage = () => ({
```

File: src/composables/useGit.js [hunk 2]
```diff
@@ -19,7 +19,6 @@ const headCommitSha = computed({
 });
 
 const getCommits = () => {
-  console.log(folderPath.value);
```
Comment (felixbouveret) [inline src/composables/useGit.js:22]: bien vu lol

File: src/modules/HomePage/components/ProjectHistoryItem/ProjectHistoryItem.vue [hunk 3]
```diff
@@ -0,0 +1,50 @@
+<template>
+  <div
+    class="bg-gray-300 rounded-lg px-4 py-2 m-2 flex items-center justify-start"
+    @click="handleClick()"
+  >
+    <svg
```
Comment (felixbouveret) [inline src/modules/HomePage/components/ProjectHistoryItem/ProjectHistoryItem.vue:6]: Maybe search for a package to directly import svg files like `vue-cli-plugin-svg`

File: src/modules/HomePage/components/ProjectHistoryItem/ProjectHistoryItem.vue [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    projectPath: {
+      type: String,
+      default: '',
+    },
+  },
+  setup(props) {
+    const { setFolderPath } = useFolder();
+    const router = useRouter();
+    const handleClick = () => {
+      console.log(props.projectPath);
```
Comment (felixbouveret) [inline src/modules/HomePage/components/ProjectHistoryItem/ProjectHistoryItem.vue:40]: left this

File: src/modules/HomePage/components/index.js [hunk 5]
```diff
@@ -0,0 +1,2 @@
+export { default as ProjectHistoryList } from './ProjectHistoryList';
+export { default as ProjectHistoryItem } from './ProjectHistoryList';
```
Comment (felixbouveret) [inline src/modules/HomePage/components/index.js:2]:
```suggestion
export { default as ProjectHistoryItem } from './ProjectHistoryItem';
```

Comment (felixbouveret) [reply src/modules/HomePage/components/index.js:2]: Actually you don't need to export it

**Review by clemcode — APPROVED** [review]
👍

**Pull request merged**